test/beast/core/_detail_get_io_context.cpp

```diff
@@ -32,13 +32,13 @@ class get_io_context_test : public beast::unit_test::suite
         BEAST_EXPECT(get_io_context(ioc) == &ioc);
         BEAST_EXPECT(get_io_context(ioc.get_executor()) == &ioc);
         BEAST_EXPECT(get_io_context(net::make_strand(ioc)) == &ioc);
-        BEAST_EXPECT(get_io_context(net::executor(ioc.get_executor())) == &ioc);
+        BEAST_EXPECT(get_io_context(net::any_io_executor(ioc.get_executor())) == &ioc);
 #if 0
```

Comment:
is it time to delete this if0 stuff?

Reply:
I'm in favour of removing cruft - less to confuse me later.